packages/manager/apps/nasha-react/src/pages/details/[serviceName]/engagementUpToContent.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {
+  OsdsChip,
+  OsdsDivider,
+  OsdsText,
+} from '@ovhcloud/ods-stencil/components/react/';
+import { OdsThemeColorIntent } from '@ovhcloud/ods-theming';
+
+interface EngagedUpToContentProps {
+  dateEngagementUpTo: string | React.ReactNode;
+  t: (key: string) => string;
+}
+
+const EngagedUpToContent: React.FC<EngagedUpToContentProps> = ({
+  dateEngagementUpTo,
+  t,
```

Comment:
Don't pass the `t` function, just use the hook
And you could define the component inside another file : you don't have to make one component per file if there is only one other component that uses this component